content/blogs/YYYY-MM-DD-CI-CD-kestra-comparison.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Kestra vs. Popular CI/CD Tools: When to Choose an Orchestration Solution"
+description: "Learn when to to choose an orchestrator rather than a CI/CD solution"
+date: xxxxxxx
+category: Solutions
+author:
+  name: Federico Trotta
+  image: "ftrotta"
+image: /blogs/xxxxx.jpg
+---
+
+In a recent [blog](./2024-09-18-what-is-an-orchestrator.md), we've defined what an orchestrator is, what are the differences between orchestration and automation, and how orchestration can help you to automate your workflows.
+
+Now, it's time to answer the question: when should you choose an orchestrator over a CI/CD solution?

```

Comment:
Use this page to help emphasise the selling points on Kestra https://kestra.io/use-cases/platform-engineers

Reply:
![use-cases-platform-engineers](https://github.com/user-attachments/assets/18558f6f-7580-469d-8ead-55c5c979eddf)
This is also broken @wrussell1999 :)

Reply:
All the use case pages should be fixed now!